File: examples/quick_start.py
import spooky_chess as sc


def main() -> None:
    game = sc.Game.standard()
    assert game.make_move_from_san("e4")
    assert game.make_move_from_san("e5")

    print(game.turn() == sc.WHITE)
    print(game.to_fen())


if __name__ == "__main__":
    main()

File: examples/quick_start.rs
#![feature(generic_const_exprs)]
#![allow(incomplete_features)]

use spooky_chess::color::Color;
use spooky_chess::game::StandardGame;

fn main() {
    let mut game = StandardGame::standard();
    assert_eq!(game.make_move_from_san("e4"), Ok(true));
    assert_eq!(game.make_move_from_san("e5"), Ok(true));

    println!("{}", game.turn() == Color::White);
    println!("{}", game.to_fen());
}
